### ISSUE TITLE
post_processing functions

### ISSUE THREAD

**Issue by santiestrada32** [conscienceli/SeqNet#2]
Dear Li,

very interesting paper and implementation. Are you also planning to release the post-processing script implementation?

thanks

**Comment by conscienceli** [comment]
Maybe later this year. Too busy due to some deadlines.

**Comment by GinkgoX** [comment]
Dear Li,
    very interesting paper and implementation. can you give the metrics result before post_processing and after post_processing so that we can have a reference result about your provided model (without post_processing), thanks.

**Comment by conscienceli** [comment]
Hi! I think i have listed them in the paper. Which results are you referring to?

**Comment by GinkgoX** [comment]
sorry about this, I get it, thanks a lot!

**Comment by KawaiiUroboros** [comment]
> Maybe later this year. Too busy due to some deadlines.

it will be really interesting to see)

**Comment by RayOoooo** [comment]
> Maybe later this year. Too busy due to some deadlines.

@conscienceli Hi! Thank you for sharing. I surpose the post-processing is the best important part to elevate the results of A/V classification. So, when will the post-processing code be updated?

**Comment by shxz1005** [comment]
Dear Li,

very interesting paper and implementation. Are you also planning to release the post-processing script implementation?

thanks

**Comment by shxz1005** [comment]
> Maybe later this year. Too busy due to some deadlines.

Dear Li,

very interesting paper and implementation. Are you also planning to release the post-processing script implementation?

thanks

**Comment by jacobdang** [comment]
> > Maybe later this year. Too busy due to some deadlines.
> 
> @conscienceli Hi! Thank you for sharing. I surpose the post-processing is the best important part to elevate the results of A/V classification. So, when will the post-processing code be updated?

Agreed. I also believe post-processing is the most interesting part of the paper.  It would be extremely helpful if the post-processing script can be released. Thanks so much.

**Comment by shxz1005** [comment]
你好，邮件已收到，谢谢~

**Comment by XDMEiGaoDong** [comment]
@conscienceli

Dear Li ，

Thank you for sharing your insightful article/post. I have been trying to implement the post-processing techniques you mentioned, but unfortunately, I'm not achieving the desired results. I'm experiencing difficulties primarily during the skeletonization and keypoint detection processes. Could you please provide some guidance on the specific functions used in post-processing, or perhaps share some related code snippets?

Thank you in advance for your time and assistance.

Warm regards

**Comment by shxz1005** [comment]
你好，邮件已收到，谢谢~

**Comment by XDMEiGaoDong** [comment]
> Maybe later this year. Too busy due to some deadlines.

@conscienceli

Dear Li Liangzhi,

Thank you for sharing your insightful article/post. I have been trying to implement the post-processing techniques you mentioned, but unfortunately, I'm not achieving the desired results. I'm experiencing difficulties primarily during the skeletonization and keypoint detection processes. Could you please provide some guidance on the specific functions used in post-processing, or perhaps share some related code snippets?

Thank you in advance for your time and assistance.

Warm regards

**Comment by ls123546** [comment]
Dear Li,

very interesting paper and implementation. Are you also planning to release the post-processing script implementation?

thanks

**Comment by shxz1005** [comment]
你好，邮件已收到，谢谢~

**Comment by ls123546** [comment]
Dear Li,


very interesting SeqNet and implementation. Are you also planning to release the post-processing script implementation?


thanks





------------------&nbsp;原始邮件&nbsp;------------------
发件人: ***@***.***&gt;; 
发送时间: 2024年4月10日(星期三) 中午11:11
收件人: ***@***.***&gt;; 
抄送: ***@***.***&gt;; ***@***.***&gt;; 
主题: Re: [conscienceli/SeqNet] post_processing functions (#2)














 你好，邮件已收到，谢谢~

**Comment by HwangYej1** [comment]
Dear Li,

very interesting paper and implementation. Are you also planning to release the post-processing script implementation?

thanks

**Comment by shxz1005** [comment]
你好，邮件已收到，谢谢~
